docker/runit/service/spark/run

```diff
@@ -76,6 +72,19 @@ function configure_properties() {
         add_if_non_empty spark.mesos.dispatcher.driverDefault.spark.executorEnv.VIRTUAL_NETWORK_ENABLED "${VIRTUAL_NETWORK_ENABLED}"
     fi
 
+    if [[ "${MESOS_ALLOCATION_ROLE}" != "" ]] && [[ "${MARATHON_APP_ENFORCE_GROUP_ROLE}" = "TRUE" ]]; then
```

Comment:
~Where does `MARATHON_APP_ENFORCE_GROUP_ROLE` come from? Can't seem to find it anywhere else in the code~
nvm, found it in the Marathon code [here](https://github.com/mesosphere/marathon/blob/master/src/main/scala/mesosphere/mesos/TaskBuilder.scala)